internal/ast/utilities.go

```diff
@@ -1290,6 +1314,37 @@ func IsThisParameter(node *Node) bool {
 	return IsParameter(node) && node.Name() != nil && IsThisIdentifier(node.Name())
 }
 
```

Comment:
a lot of this syntax isn't supported, or supported yet, in Corsa. Merge fom main and use GetAssignmentDeclarationKind as-is.